preview-env/create/action.yml

```diff
@@ -133,3 +133,19 @@ runs:
       env_url: ${{ inputs.app_url }}
       ref: ${{ inputs.revision }}
       env: ${{ inputs.app_name }}
+  - name: Create deployment status artifact
+    if: always()
+    shell: bash
+    run: |
+      if [[ "${{ github.action_status }}" == "success" ]]; then
+        echo ":full_moon_with_face: [\`${{ inputs.app_name }}\`](${{ inputs.app_url }}): ${{ github.action_status }}\n" > status_${{ inputs.app_name }}.md
+      else
+        echo ":boom: \`${{ inputs.app_name }}\`: ${{ github.action_status }} :arrow_right: see [ArgoCD](https://${{ inputs.argocd_server }}/applications?health=Progressing%2CDegraded&search=${{ inputs.app_name }})\n" > status_${{ inputs.app_name }}.md
+      fi
+  - name: Upload deployment status as artifact
+    if: always()
+    uses: actions/upload-artifact@v4
+    with:
+      name: deployment-status-${{ github.run_id }}-${{ github.run_attempt }}-${{ inputs.app_name }}
+      path: status_${{ inputs.app_name }}.md
+      retention-days: 1
```

Comment:
If there is no need to keep the artifact once the workflow is finished (e.g. for rerun or so), we probably can remove it ? Perhaps we can mark it as ephemeral ?

Reply:
unfortunately, there's no built-in option to mark it as ephemeral. 
That's the reason why I chose the minimum retention value.
So I judged, that it doesn't have a big footprint, hence it shouldn't hurt to keep it for a day.